gamebook/scene.py

```diff
@@ -35,20 +35,24 @@ def show_options(self):
         return choice_list
 
     def get_option_by_user_input(self,user_input_recieved):
-        option_to_return = Option("",[],[])
         for option in self.options:
             for user_input in option.user_input:
```

Comment:
Maybe rename `Option.user_input` to `Option.user_inputs` because it takes multiple values possibly.